hypha/apply/funds/forms.py

```diff
@@ -208,7 +198,7 @@ def clean_submissions(self):
 class UpdateReviewersForm(ApplicationSubmissionModelForm):
     reviewer_reviewers = forms.ModelMultipleChoiceField(
         queryset=User.objects.reviewers().only("pk", "full_name"),
-        widget=Select2MultiCheckboxesWidget(attrs={"data-placeholder": "Reviewers"}),
+        widget=Select2MultiCheckboxesWidget(attrs={"data-placeholder": "Select..."}),
```

Comment:
Can we not delete all the ` widget=Select…` lines now that we use choice.js?

Reply:
Yes, I think we can delete these.